### PR TITLE
First design iteration for adding results in the madlib style

### PR DESCRIPTION
**Pull request by mborsare**
screen shot 2017-07-06 at 12 30 31 pm

https://trello.com/c/kNpbiQHP

### CODE REVIEW

**Comment by mborsare** [comment]
screen shot 2017-07-06 at 1 09 07 pm

**Comment by mborsare** [comment]
@ryancoughlin updated

**Comment by mborsare** [comment]
screen shot 2017-07-06 at 1 11 46 pm

File: app/assets/stylesheets/components/_headline-poster.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@
   font-family: $heading-font-family;
   font-size: modular-scale(2, $base-font-size);
   font-weight: normal;
+  hyphens: auto;
```
Comment (ryancoughlin) [inline app/assets/stylesheets/components/_headline-poster.scss]: TIL

**Pull request merged**